test/kitchen/docs/README.md

```diff
@@ -1,42 +1,128 @@
 
-##### Test execution
+#### testkitchen_deploy
 
-Each individual test environment is controlled from _.gitlab\_ci.yml_.  The tests are set up using an environment variable, which is a list of test platforms which are turned into test kitchen platforms.  In _.gitlab\_ci.yml_, the platform list has the following syntax:
-        `short_name1,azure_full_qualified_name1|short_name2,azure_full_qualified_name2`
+This stage contains jobs to deploy the Agent packages created by the package build jobs to our test repositories:
+- apttesting.datad0g.com (for .deb packages used by the Debian and Ubuntu tests)
+- yumtesting.datad0g.com (for .rpm packages used by the CentOS/RHEL and SUSE tests)
+- s3.amazonaws.com/dd-agent-mstesting (for .msi packages used by the Windows tests)
 
-The azure qualified name for an image can be found by using `az vm image list`  to search for the image (using the publisher name, the offer name and/or the sku). The _urn_ field contains the azure qualified name.
+For each package type, there are two jobs: one to deploy the A6 package, the other
+to deploy the A7 and IoT packages.
 
-This configuration is fed to the script _tasks/run-test-kitchen.sh_. That script in turn sets up additional variables for Azure (authentication/environment variables), and then executes the actual kitchen command.  Using the ruby _erb_ syntax, a full matrix of platforms and tests is generated and executed.
+Each pipeline puts its packages in a dedicated branch:
+- .deb packages are put in the `dists/pipeline-<PIPELINE_ID>-a6` and `dists/pipeline-<PIPELINE_ID>-a7` branches apttesting.datad0g.com.
+- .rpm packages are put in the `pipeline-<PIPELINE_ID>-a6` and `pipeline-<PIPELINE_ID>-a7` branches of yumtesting.datad0g.com.
+- SUSE .rpm packages are put in the `suse/pipeline-<PIPELINE_ID>-a6` and `suse/pipeline-<PIPELINE_ID>-a7` branches of yumtesting.datad0g.com.
+- .msi packages are put in the `pipelines/A6/<PIPELINE_ID>` and `pipelines/A7/<PIPELINE_ID>` branches of s3.amazonaws.com/dd-agent-mstesting.
 
-There are currently two input files.
-1. _kitchen-azure.yml_ contains a set of installation and upgrade tests that are executed on all of the available platforms
-2. _kitchen-azure-winstall.yml_ contains a set of  Windows specific tests for command-line installation options. These tests are currently (as configured in _.gitlab-ci.yml_) only run on one Windows OS for brevity.
+#### testkitchen_testing
+
+This stage contains jobs which execute the kitchen tests on our Azure account.
+
+Each job executes one specific test suite, for one OS, for one Agent flavor and one major Agent version.
+
+The `.gitlab-ci.yml` file has been structured in a way to avoid config duplication:
```

Comment:
nit: the wording sounds a bit weird to me, maybe:
```suggestion
The `.gitlab-ci.yml` file has been structured to avoid config duplication with the following jobs:
```